examples/fluid/wetting_water_spheres_2d.jl

```diff
@@ -1,13 +1,13 @@
-# In this example two circles of water drop to the floor demonstrating the difference
-# between the behavior with and without surface tension modelling.
+# In this example we try to approach the static shape of a water droplet on a horizontal plane.
+# The shape of a static droplet can be calculated from the Young-Laplace equation.
```

Comment:
Can you `trixi_include` the falling spheres example? They look very similar to me.